src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,48 +3,175 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Node<T> first;
+
```

Comment:
it is better to implement 2 fields that will point to the head (first) and tail (last). 
private Node<T> head;
 private Node<T> tail;